client/src/app/all-projects/all-projects.component.ts

```diff
@@ -3,10 +3,12 @@ import {MatPaginator} from '@angular/material/paginator';
 import {Project, ProjectColumn} from '../shared/models/project';
 import {IngestService} from '../shared/services/ingest.service';
 import {map} from 'rxjs/operators';
-import {Criteria} from '../shared/models/criteria';
 import {MetadataDataSource} from '../shared/data-sources/metadata-data-source';
 import {PagedData} from '../shared/models/page';
 import {MetadataDocument} from '../shared/models/metadata-document';
+import * as ingestSchema from '../project-form/project-ingest-schema.json';
```

Comment:
I would organise the imports alphabetically as well as group by library imports and our own imports. the IDE can do it for you.